tools/gsc-mannwhitney_de/mannwhitney_de.xml

```diff
@@ -0,0 +1,94 @@
+<tool id="mannwhitney_de" name="Perform a differential analysis" version="0.1.0">
+    <description>between the two groups of a signature score using a Mann-Whitney test</description>
```

Comment:
`between the two groups of a signature score using a Mann-Whitney test`
-->
`using a Mann-Whitney test`